setup.cfg

```
@@ -9,6 +9,7 @@ markers =
     only_with_platform: DEPRECATED only used for legacy tests. Use skipMlFramework instead. marks a test to be performed only for a specific mlFramework values
     framework_agnostic: marks a test to be agnostic to frameworks and run only for one default framework
     skip_travis: Skips a test marked with this decorator if the command line argument skip_travis is set to true
+    skipModule: Skip the test if a module is not available in the current environment
```

How about adapting the marker name following the existing naming pattern? Although `skipMlFramework` is also an outlier.
```suggestion
    skip_module: Skip the test if a module is not available in the current environment
```

---

The original code to skip modules uses camel case. I agree that both `skipModule` and `skipMlFramework` are outliers. So let's start to fix the `skip_module` with this PR :relaxed:

I will open an issue for `skipMlFramework` and if time allows update it as well :+1: